prediction_market_agent/agents/microchain_agent/prompts.py

```diff
@@ -7,7 +7,8 @@
 
 {{engine_help}}
 
-Only output valid Python function calls, without code formatting characters.
+Only output valid Python function calls, without code formatting characters, without any other text.
```

Comment:
Small changes needed to make it work with Llama 3.1. I think it shouldn't affect Openai models in any way, it just clarifies the output format.